src/torrent_handle.cpp

```diff
@@ -653,6 +653,11 @@ namespace libtorrent {
 		return sync_call_ret<bool>(false, &aux::torrent::user_have_piece, piece);
 	}
 
+    void torrent_handle::set_sequential_start(piece_index_t start_piece, piece_index_t end_piece) const
```

Comment:
it would probably make more sense to call this `set_sequential_range()`.

Also, what if this range is invalid? (`end_piece` < `start_piece` or `start_piece` < 0).

There should probably be a `TORRENT_ASSERT_PRECOND()` but also a check and just ignore the call in that case.